vm.c

```diff
@@ -2137,6 +2137,22 @@ rb_iseq_eval(const rb_iseq_t *iseq)
     return val;
 }
 
+VALUE
+rb_iseq_eval_in_scope(const rb_iseq_t *iseq, VALUE scope)
+{
+    rb_execution_context_t *ec = GET_EC();
+    rb_binding_t *bind = Check_TypedStruct(scope, &ruby_binding_data_type);
+
+    vm_set_eval_stack(ec, iseq, NULL, &bind->block);
+
+    /* save new env */
+    if (iseq->body->local_table_size > 0) {
+      vm_bind_update_env(scope, bind, vm_make_env_object(ec, ec->cfp));
```

Comment:
The indent is slipped.

Reply:
My apologies for the style error, I've fixed this.